front/app/containers/MainHeader/Components/LanguageSelector/index.tsx

```diff
@@ -157,10 +169,11 @@ const LanguageSelector = ({
           />
         </DropdownButton>
 
-        <Dropdown
+        <StyledDropdown
+          className={className}
           width="180px"
           mobileWidth="160px"
-          top={top || '68px'}
+          top={useDefaultTop ? top || '68px' : top || undefined}
           right={right ? right : !isRtl ? '0px' : undefined}
```

Comment:
In order for the "open-upwards" behaviour to work, we need to actually set top to "undefined". However, we're currently using a default fallback here, so I needed to add a new attribute to disable the fallback for this LanguageSelector case.